### ISSUE TITLE
Support for socket.io library

### ISSUE THREAD

**Issue by johnhigginsavila** [Mackignacio/maya#13]
## 📝 Description
A lot of projects relies on socket.io for realtime communications, can we add a wrapper to support socket.io?

## 📋 Todo:

- add a socket.io wrapper and necessary dependencies
- add unit testing
- update documentation for socket.io wrapper

**Comment by mackignacio** [comment]
Yes we can surely add this feature in version 1.0.0. Right now our main priority is the Unit testing of existing code and creating the CLI.
